### PR TITLE
1.1.0 Release

### PR DESCRIPTION
**Pull request by nblumhardt**
* #10 - (sample) - new date/time text parser example
 * #11 - RTM `dotnet` tooling
 * #16 - prevent large .NET Standard package surface area being pulled into .NET Framework projects
 * #17 / #14 - report expectations correctly when `Span.EqualTo()` hits EOF

### CODE REVIEW

**Comment by nblumhardt** [comment]
@KodrAus I think this one is ready to go

File: sample/DateTimeTextParser/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Superpower sample / DateTimeTextParser
+
+This example should shows how to build a simple text parser with Superpower.
+It uses a simple and well known requirement: parsing date and time values 
+according to ISO-8601 format.
+
+## The requirement
+
+In a simple, custom query language a user enter multiple conditions where a
```
Comment (KodrAus) [inline sample/DateTimeTextParser/README.md]: Just a few typos in here.

File: sample/DateTimeTextParser/DateTimeParser.csproj [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="superpower" Version="1.0.2" />
```
Comment (KodrAus) [inline sample/DateTimeTextParser/DateTimeParser.csproj]: Could this be a project reference? (more along the lines of the `IntCalc` csproj)

Comment (nblumhardt) [reply sample/DateTimeTextParser/DateTimeParser.csproj]: Nice spotting 👍

**Pull request merged**